op.c

```diff
@@ -10723,6 +10724,32 @@ Perl_newDEFEROP(pTHX_ I32 flags, OP *block)
     return o;
 }
 
+/*
+=for apidoc op_wrap_finally
+
+Wraps the given C<block> optree fragment in its own scoped block, arranging
+for the C<finally> optree fragment to be invoked when leaving that block for
+any reason. Both optree fragments are consumed and the combined result is
+returned.
+
+ */
```

Comment:
This extra whitespace before the closing `*/` causes a warning when compiling:
```
=cut missing? op.c:10735: */
```

Reply:
Fixed in c33a5ad6f8